src/core/agent/services/identifierService.ts

```diff
@@ -375,6 +377,86 @@ class IdentifierService extends AgentService {
     });
     return multisigId;
   }
+
+  async rotateMultisig(ourIdentifier: string): Promise<string | undefined> {
+    const metadata = await this.getMetadataById(ourIdentifier);
+    if (metadata.method !== IdentifierType.KERI) {
+      throw new Error(IdentifierService.ONLY_CREATE_ROTATION_WITH_AID);
+    }
+    if (!metadata.signifyName) {
+      throw new Error(IdentifierService.AID_MISSING_SIGNIFY_NAME);
+    }
+    // Make sure no non-Keri contacts get passed into this function
+    const members = await this.agent.modules.signify.getMultisigMembers(
+      metadata.signifyName
+    );
+    const multisigMembers: Aid[] = [];
+    await Promise.all(
+      members?.signing?.map(async (signing: any) => {
+        const aid = await this.agent.modules.signify.queryKeyState(
+          signing.aid,
+          "0"
```

Comment:
I think it is ok for now, because when I use the undefined sequence number, is cannot find the state

Reply:
We need to get the correct state. Otherwise, it's not correct. Is there anything in the signing members response to tell us the index used previously?

Reply:
Nope, in signing  members don't have it. It is no way to know it now I guess

Reply:
It needs to be exchanged out of band, then. What happens if you try to rotate this twice? Do all keys update?

Reply:
You are right about the sequence number, I add more logic to check that and tested. We can rotate multi time now.